test/e2e/opaque_resource.go

```diff
@@ -38,7 +38,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = framework.KubeDescribe("Opaque resources [Feature:OpaqueResources]", func() {
+var _ = framework.KubeDescribe("Opaque resources [Serial]", func() {
```

Comment:
Without serial, this group would run in parallel to other groups. And I dont see this group affecting other groups when run in parallel.